OsmAnd/res/values/strings.xml

```diff
@@ -73,7 +73,7 @@
     <string name="app_mode_subway">Subway</string>
     <string name="app_mode_horse">Horse</string>
     <string name="app_mode_helicopter">Helicopter</string>
-    <string name="osmand_routing_promo">You can add your own modified version of the file routing.xml in ..osmand/routing</string>
+    <string name="osmand_routing_promo">Optionally modify routing.xml in ..osmand/routing</string>
```

Comment:
Is this meant to be "..osmand"?

Reply:
Yes, it is the folder name. And I would leave in the word 'file' because this really relates to a file modification.